README.md

```diff
@@ -96,16 +96,10 @@ devcontainer build --push true --workspace-folder . --platform="linux/amd64,linu
 <details>
 <summary>How to Run Inside ISAAC ROS Container/Dev Container On Linux/Jetson</summary>
 <br>
-First, do the following before running run_dev.sh:
+First, do the following before running:
+    
+Install nvidia ngc cli and make sure it is present in path
 
-```
-printf "CONFIG_IMAGE_KEY=ros2_humble.realsense.deepstream.user.zed.umn \n" > ~/Lunabotics/src/isaac_ros/isaac_ros_common/scripts/.isaac_ros_common-config 
-``` 
-To use Gazebo in the ISAAC ROS container, do this instead:
-
```

Comment:
With how often this changes this may go obsolete, so I wouldn't do to many details. We also can cache these so only people who build base images need to perform this step.

Reply:
The README should include everything that new people need to know to get setup / run things. It would be awesome if we can cache these so that only 1 or 2 people need to worry about doing it

Reply:
I think the only instructions we need to add are installing nvcr cli and also state that whenever the repo is freshly cloned people need to add a 'deepstream' folder to the 'docker' folder, right?